Cargo.toml

```diff
@@ -10,6 +10,6 @@ categories = ["development-tools"]
 [dependencies]
 rustc-serialize = "0.3"
 log = "0.3"
-rls-data = "= 0.4.1"
+rls-data = { path = "../rls-data", default-features = true, features = ["borrows"] }
```

Comment:
I misread the docs the first time. I thought it was necessary when overriding the `features` list, but it seems to not be the case. And yes, this is dependent on both rustc and rls-data. Did I mention that incorrectly in the rust-lang/rust PR?